.github/workflows/build-containers.yml

```diff
@@ -46,6 +47,13 @@ jobs:
           username: flowforge
           password: ${{ secrets.DOCKER_HUB_PASSWORD }}
           readme-filepath: docker-compose/README.md
+      - name: Push README flowfuse
+        uses: peter-evans/dockerhub-description@v3
+        with:
+          repository: flowfuse/forge-docker
+          username: flowforge
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
```

Comment:
The `flowfuse` org has a different set of credentials

Reply:
flowforge user is a collaborator on all flowfuse containers so should be able to push to them